lib/desired-caps.js

```diff
@@ -75,6 +75,9 @@ let desiredCapConstraints = _.defaults({
   wdaStartupRetryInterval: {
     isNumber: true
   },
+  prebuildWDA: {
```

Comment:
do you think there's any potential name confusion here between this cap and `usePrebuiltWDA`?